_integration-tests/tests/civisibility/main_test.go

```diff
@@ -0,0 +1,159 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+//go:build integration
+
+package civisibility
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/tinylib/msgp/msgp"
+)
+
+var ciVisibilityPayloads mockPayloads
+
+func TestMain(m *testing.M) {
```

Comment:
Optional, but it might be a good idea to validate that Orchestrion did indeed instrument the package (for otherwise, you'll get a ton of assertion failures that are actually just "there is no instrumentation here").

There's a magic comment you can use to have a package-level `const` (or `var` but I'd recommend the `const` form here) boolean variable turn to `true` when build w/ orchestrion.

See: https://datadoghq.dev/orchestrion/docs/built-in/directive/orchestrion-enabled/

Reply:
oh that's nice, done in [92ee768](https://github.com/DataDog/orchestrion/pull/340/commits/92ee768360c07a61c2311485193b2236c93be0b1)